app/services/github_service.rb

```diff
@@ -28,8 +28,8 @@ def load_repositories(access_token)
       # Add repos and remove user from any repos they no longer have access to
       remove_outdated(user, outdated_repos)
     end
-
-    def submit_issue(repo_id, sub_name, email, title, details)
+    
+    def submit_issue(repo_id, sub_name, email, email_public, title, details)
```

Comment:
Seems to be unsuccessful at https://github.com/schneidmaster/gitreports.com/pull/98/commits/1710441b4e1c21bc6e9fd462d2cd786b32a1ad2d